### PR TITLE
Update validator version & fix tests

### PR DESCRIPTION
**Pull request by sebastianbenz**
We should update the AMP validator dependency to 1.30.0 as well.

### CODE REVIEW

File: validator/gulpjs/test/validate.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ describe('gulp-amphtml-validator', function() {
       const validFile = createFile(VALID_FILE);
       validate.write(validFile);
       validate.once('data', function(file) {
-        assert.equal(file.ampValidationResult.status, 'N/A');
+        assert.equal(file.ampValidationResult.status, 'UNKNOWN');
```
Comment (honeybadgerdontcare) [inline validator/gulpjs/test/validate.js]: Curious, why was this `N/A`?

Comment (honeybadgerdontcare) [reply validator/gulpjs/test/validate.js]: Nevermind, I see it below now.

**Comment by sebastianbenz** [comment]
@honeybadgerdontcare can you publish the module to NPM?

**Comment by honeybadgerdontcare** [comment]
@sebastianbenz done

**Comment by sebastianbenz** [comment]
Great - thanks a lot!

>

**Pull request merged**